modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -427,6 +429,32 @@ void ONNXImporter::addLayer(LayerParams& layerParams,
     }
 }
 
+void ONNXImporter::expandMid(const LayerParams& layerParams, opencv_onnx::NodeProto& node_proto,
+                             const std::string& input, size_t n, int axis)
+{
+    String base_name = layerParams.name + "/copy_";
+    std::vector<std::string> input_names;
+    for (int j = 0; j < n; j++)
+    {
+        std::ostringstream ss;
+        ss << j;
+        LayerParams copyLP;
+        copyLP.name = base_name + ss.str();
+        copyLP.type = "Identity";
+        CV_Assert(layer_id.find(copyLP.name) == layer_id.end());
```

Comment:
`CV_CheckEQ` with a good error message?

Reply:
It supports only int, float and double. Can you recommend any other macro?

Reply:
This case is not supported by CV_Check* macros yet. It makes sense to keep CV_Assert() here for now.

Reply:
` CV_Assert((layer_id.find(copyLP.name) == layer_id.end()) && "Explanatory message");`?

Reply:
@VadimLevin, done